pom.xml

```diff
@@ -29,11 +36,22 @@
     <name>WildFly Elytron</name>
     <description>WildFly Security SPIs</description>
 
-    <parent>
-        <groupId>org.jboss</groupId>
-        <artifactId>jboss-parent</artifactId>
-        <version>16</version>
-    </parent>
+    <issueManagement>
+        <system>Jira</system>
+        <url>https://issues.jboss.org/browse/ELY</url>
+    </issueManagement>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <scm>
```

Comment:
I'm not in favor of an scm block because project hosting is a moment-in-time trait, whereas the tag will last indefinitely.